variables.tf

```diff
@@ -233,3 +233,9 @@ variable "backup_crn" {
     error_message = "backup_crn must be null OR starts with 'crn:' and contains ':backup:'"
   }
 }
+
+variable "member_host_flavor" {
+  type        = string
+  description = "Allocated host flavor per member. For more information, see https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs/resources/database#host_flavor"
+  default     = null
+}
```

Comment:
When adding variables to the main module also add them to sub modules, especially fscloud (redis has this, so it needs updating) and any DAs (redis does not have one of these).
postgres also shows how to add this to fscloud sub module.